packages/web-components/src/components/masthead/megamenu-top-nav-menu.ts

```diff
@@ -26,16 +29,16 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
 @customElement(`${ddsPrefix}-megamenu-top-nav-menu`)
 class DDSMegaMenuTopNavMenu extends DDSTopNavMenu {
   /**
-   * The menu ul node.
+   * The megamenu component
    */
-  @query(`.${prefix}--header__menu`)
-  private _menuNode!: HTMLElement;
+  @property()
+  megaMenu!: DDSMegaMenu;
 
   /**
-   * The trigger button.
+   * The menu ul node.
    */
-  @query('[part="trigger"]')
-  private _topMenuItem!: HTMLAnchorElement;
+  @query(`.${prefix}--header__menu`)
+  private _menuNode!: HTMLElement;
 
```

Comment:
```suggestion
  /**
   * The trigger button.
   */
  @query('[part="trigger"]')
  private _topMenuItem!: HTMLAnchorElement;
```

Reply:
```
/**
   * The menu ul node.
   */
  @query(`.${prefix}--header__menu`)
  private _menuNode!: HTMLElement;
```

is what I see currently from the `main` branch, was there a change in the `feat/cwc-v2` branch for megamenu?